Example/index.android.js

```diff
@@ -5,12 +5,7 @@
 'use strict';
 
 import React from 'react';
-import ReactNative from 'react-native';
```

Comment:
Are there any significant changes in this file that need to be reviewed? Most of them look like Formatting changes.

Reply:
The default export of react-native has been removed from web and will be removed from native soon.